src/Appwrite/Platform/Workers/Deletes.php

```diff
@@ -186,7 +184,6 @@ private function deleteSchedules(Database $dbForConsole, callable $getProjectDB,
             'schedules',
             [
                 Query::equal('region', [App::getEnv('_APP_REGION', 'default')]),
-                Query::equal('resourceType', [$document->getAttribute('resourceType')]),
```

Comment:
I don't think we need `deleteSchedule`, since there's no additional resources to delete. I think it will be better to immediately delete the schedule from the messaging worker, instead of updating it to inactive, then cleaning up here.